mappings/net/minecraft/client/gui/widget/WidgetLock.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS none/bew net/minecraft/client/gui/widget/WidgetLock
```

Comment:
Should probably be `WidgetButtonLock` to keep in line with `WidgetButtonLanguage`, since they both just render a special texture over the standard button.